packages/components/src/ResourcePicker/Toolbar/StateFilter/StateFilter.component.js

```diff
@@ -25,13 +26,13 @@ function StateFilter({ t, types, onChange, selected, favorites, certified }) {
 				<span className={classNames(theme['option-label'])}>
 					{t('FILTER', { defaultValue: 'Filter:' })}
 				</span>
-				{types.includes(TYPES.SELECTED) && (
+				{types.includes(TYPES.SELECTION) && (
 					<ActionIconToggle
 						icon={'talend-check-circle'}
-						label={t('SELECTED', { defaultValue: 'Selected' })}
-						active={selected}
-						onClick={() => onChange(TYPES.SELECTED, !selected)}
-						className={classNames(theme['tc-resource-picker-selected-filter'])}
+						label={t('SELECTION', { defaultValue: 'Selected' })}
```

Comment:
why replace the translation key? and more genrally why replace selected by selection?